news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – May 2020 edition"
+subtitle: "Where you'll learn that the Swahili translation project is waiting for you"
+date:  2020-06-12 14:30:11
+authors: [ JuneByun ]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, Swahili will be available in PrestaShop 1.7.8! :tada:
+
+I have the pleasure to announce that PrestaShop will finally be available in Swahili in the 1.7.8 version! I thus encourage you to translate, suggest, and proofread the PrestaShop Swahili 1.7.7 translation project. Currently, there aren’t many Swahili translators on PrestaShop’s Crowdin community, so if you know any Swahili natives, spread the word and do not hesitate to invite them to participate in the [PrestaShop Swahili translations project](https://crowdin.com/project/prestashop-official/sw#), they will be more than welcome! :heart_eyes:
```

Comment:
I find this a little confusing:

> PrestaShop will finally be available in Swahili in **the 1.7.8 version**!
> I thus encourage you to translate, (...) the PrestaShop Swahili **1.7.7 translation project**

Reply:
@eternoendless thanks for the comment! I'll simply take off 1.7.7 😉